components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.common/src/main/java/org/wso2/carbon/identity/api/server/application/management/common/ApplicationManagementConstants.java

```diff
@@ -132,6 +132,9 @@ public enum ErrorMessage {
         INVALID_POLICY_TYPE_FOR_API_RESOURCE("60512", "Invalid policy type provided for the API " +
                 "resource.", "API resource with id: %s doesn't allow the provided policy type: %s."),
 
+        ERROR_ASSERT_LOCAL_SUBJECT_IDENTIFIER_DISABLED("60513",
+                "Invalid configuration.",
+                "useMappedLocalSubject cannot be disabled when mappedLocalSubjectMandatory is enabled."),
```

Comment:
let's enclose the config names with quotes for better readability

Reply:
addressed with https://github.com/wso2/identity-api-server/pull/521/commits/c55442cff0175c0b4007f2803256c71b8bcaea36